bundles/WebToPrintBundle/src/Installer.php

```diff
@@ -26,16 +28,34 @@ class Installer extends SettingsStoreAwareInstaller
         'web2print_settings',
     ];
 
+    protected const STANDARD_DOCUMENT_ENUM_TYPES = [
+        'page',
+        'link',
+        'snippet',
+        'folder',
+        'hardlink',
+        'email',
+        'newsletter'
+    ];
+
+    protected const BUNDLE_EXTRA_DOCUMENT_ENUM_TYPES = [
+        'printpage',
+        'printcontainer'
+    ];
+
     public function install(): void
     {
         $this->installDatabaseTable();
+        $enums = array_merge(self::STANDARD_DOCUMENT_ENUM_TYPES, self::BUNDLE_EXTRA_DOCUMENT_ENUM_TYPES);
+        $this->modifyEnumTypes($enums);
```

Comment:
if there's a bundle that also adds new document types, it won't work properly, or at least it would all depend on the installation order/priority, right?

For instance, headlessdocuments
https://github.com/pimcore/headless-documents/blob/bc6884c65ef3c30ea516afc5327226e96feeffed/src/Installer.php#L56

Could we instead use something that adds/remove items from existing array?

Reply:
You are totally right, I will check that.

Reply:
But then we have to change it for the headless documents too right? 
Because if you install it, printpage and printcontainer will be added never the less?